Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -246,7 +246,7 @@ div.bocList table.bocListTable tr.bocListDataRow.selected td.bocListDataCell
   background-color: var(--color-highlight);
 }
 
-div.bocList table.bocListTable td.bocListDataCell
+div.bocList table.bocListTable tr:not(.hasValidationRow) td.bocListDataCell
```

Comment:
We also have the `NovaViso` theme, which should probably? also get the validation functionality so we need to add CSS for that as well. To toggle the theme look at the `NovaViso`/`NovaGray` classes

Reply:
Nah, I don't think NovaGray needs validation. It's good as is, no need to change/add anything.

Reply:
All choking (sic) aside, there's a couple of situation where we actually do skip something in NovaGray, so sarcasm about not doing NovaGray isn't always obvious (depends on just how much sarcasm/irony drips from the wording). In particular, for our primary usecase, the new validation will only get used in applications that use NovaViso. So, it could actually be a valid discussion if we should skip NovaGray. The basic rule is: is it cheap, then we cover NovaGray, too. Is it expensive, we need to discuss it.

Reply:
(NovaGray is backwards compatiblity support only)